test/test_plugin_twitter.py

```diff
@@ -585,12 +657,17 @@ def test_plugin_twitter_dm_attachments(mock_get, mock_post):
         os.path.join(TEST_VAR_DIR, 'apprise-test.mp4'),
     ]
 
+    # Create application objects.
+    obj = Apprise.instantiate(twitter_url)
+
     assert obj.notify(
         body='body', title='title', notify_type=NotifyType.INFO,
         attach=attach) is True
 
+    # No get request as cached response is used.
+    # TODO: Review this.
     assert mock_get.call_count == 0
```

Comment:
This will fail when being invoked with `pytest test/test_plugin_twitter.py -nauto`.

Reply:
Dito.